Sources/Sentry/SentryANRTrackingIntegration.m

```diff
@@ -15,53 +23,38 @@
 
 #if SENTRY_HAS_UIKIT
```

Comment:
`m`: I think we can remove this and make ANRs available on all platforms. Same applies to the tests.